packages/core/addon/components/pick-container.js

```diff
@@ -18,11 +18,16 @@
  *      {{/pick-form}}
  *   {{/pick-container}}
  */
-import Ember from 'ember';
+import { assert } from '@ember/debug';
+import { assign } from '@ember/polyfills';
+import { copy } from 'ember-copy';
+import { get, computed, observer } from '@ember/object';
+import { typeOf } from '@ember/utils';
+import $ from 'jquery';
```

Comment:
can we make a ticket to get rid of jquery here? I don't think it's necessary.

Reply:
Let's handle removal of jquery in a separate task.

Reply:
yeah ask was to make a ticket for it so we can tackle it separately.